hosting/src/components/Form/Dropzone.tsx

```diff
@@ -0,0 +1,133 @@
+"use client";
+import "@/components/Form/styles/dropzone.scss";
+import {getAcceptDescription, isFileAccepted} from "@/utils/fileUpload";
+import {AcceptFileType} from "@functions/definitions/AcceptFileType";
+import React, {useEffect, useRef, useState} from "react";
+
+// Props interface for the Dropzone component
+export interface DropzoneProps {
+  value?: string;
+  disabled?: boolean;
+  accept?: AcceptFileType;
+  onChange?: (fileString: string | null, fileBlob: File | null) => void; // Callback to send the file data to parent
+}
+
+/**
+ * Dropzone component for handling file uploads and sending data to parent without preview.
+ * @param {DropzoneProps} props - The properties for the dropzone component.
+ * @return {JSX.Element} The rendered dropzone component.
+ */
+export function Dropzone({value, disabled, accept = AcceptFileType.AllFiles, onChange}: DropzoneProps): JSX.Element {
+  const [dragActive, setDragActive] = useState(false); // State to track if a file is being dragged over the dropzone.
+  const inputRef = useRef<HTMLInputElement>(null); // Reference to the hidden file input element.
+
+  /**
+   * useEffect hook that resets the file input value when the component unmounts.
+   */
+  useEffect(() => {
+    return () => {
+      if (inputRef.current) {
+        // Check if the input element exists.
+        inputRef.current.value = ""; // Reset the input value to an empty string.
+      }
+    };
+  }, [value]); // This effect depends on the `value` prop.
```

Comment:
Clean up chatgpt generated comments 🙏  (check all files and places)

Reply:
Done ✅